src/ngraph/runtime/cpu/cpu_external_function.hpp

```diff
@@ -72,18 +74,40 @@ namespace ngraph
 
             using OpMap = std::unordered_map<std::type_index, OpFunction>;
 #endif
+            struct TensorTracerAttributes
+            {
+                size_t number_of_elements;
+                ngraph::Shape shape;
+                element::Type type_of_element;
+
+                TensorTracerAttributes(const size_t size,
+                                       const ngraph::Shape& shape,
+                                       const element::Type& type)
+                    : number_of_elements(size)
+                    , shape(shape)
+                    , type_of_element(type)
+                {
+                }
+            };
 
             struct OpAttributes
             {
                 std::string Description;
                 std::vector<std::string> Outputs;
                 std::vector<std::string> Inputs;
+                std::vector<TensorTracerAttributes> Outputs_tensor_attrs;
```

Comment:
Follow the coding standards.

Reply:
it should be lower case?(why other variable names here with uppercase)?